src/development/ui/material.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Material Design for Flutter
+description: Learn about Material Design for Flutter.
+---
+
+Material is a design system created by Google to
+help teams build high-quality digital experiences
+for Android, iOS, Flutter, desktop, and the web.
+
+Material 3 is the latest generation of Material Design.
+Flutter is in the process of migrating to Material 3.
+
+{{site.alert.note}}
+  A few facts about Material 3 (M3),
+  the next generation of Material Design:
+
+  * Most Flutter widgets are already migrated to M3,
+    but you can follow progress on the
+    [Material 3 Flutter GitHub project][] and the
+    [GitHub umbrella issue][].
+  * For the latest list of Flutter's widgets that
+    are new and migrated to Material 3, see
+    [Affected widgets][].
+  * Until migration is complete, you can opt in to
+    Material 3 using the [`useMaterial3`][] property.
+    However, your UI might be inconsistent until
+    _all_ of Flutter _and_ your code is migrated.
+  * Some Flutter widgets have similar behavior in M2 and
```

Comment:
"The vast majority of Flutter widgets..."

Reply:
Done